subjects/unwrap_or_expect/README.md

```diff
@@ -53,38 +39,27 @@ Here is a program to test your function:
 use unwrap_or_expect::*;
 
 fn main() {
-    // // if uncommented, the below line will give an expect "ERROR "
-    // println!("{:?}", expect(vec![1, 3, 2, 5]));
-
-    println!("{:?}", unwrap_or(vec![1, 3, 2, 5]));
-    println!("{:?}", unwrap_or(vec![1, 3, 5]));
-
-    println!("{:?}", unwrap_err(vec![1, 3, 2, 5]));
-
-    // // if uncommented, the below line will give an unwraped error
-    // println!("{:?}", unwrap_err(vec![1, 3, 5]));
+    println!("{}", fetch_data(Ok("server1.com".to_string()), Security::Medium));
+    println!("{}", fetch_data(Err(String::new()), Security::Medium));
+    println!("{}", fetch_data(Err("server2.com".to_string()), Security::Low));
 
-    println!("{:?}", unwrap(vec![1, 3, 5]));
+    // Panics with no custom message
+    // fetch_data(Err("ERROR CRITICAL".to_string()), Security::Unknown);
 
-    //// if uncommented, the below line will give an error
-    // println!("{:?}", unwrap(vec![1, 3, 2, 5]));
+    // Panics with the message "ERROR: program stops"
+    // fetch_data(Err(String::new()), Security::High);
 
-    println!("{:?}", unwrap_or_else(vec![1, 3, 5]));
-    println!("{:?}", unwrap_or_else(vec![3, 2, 6, 5]));
+    // Panics with the message "malicious_server.com"
+    // fetch_data(Ok("malicious_server.com".to_string()), Security::BlockServer);
 }
 ```
 
 And its output:
 
 ```console
-[]
-[2, 4, 6]
-("There is a even value in the vector!", [2])
-[2, 4, 6]
-Ok([2, 4, 6])
-[2, 4, 6]
-Err(("There is a even value in the vector!", [2, 6]))
-[2, 6]
+server1.com
```

Comment:
Some other exercises have this before the output. Should we include that here also?
```
$ cargo run
```

Reply:
Yes, I've added it for consistency